apps/console/src/components/pages/protected/evidence/evidence-create-form.tsx

```diff
@@ -0,0 +1,338 @@
+                          </FormLabel>
+                          <FormControl>
+                            <Popover open={isCreationDateCalendarOpen} onOpenChange={setIsCreationDateCalendarOpen}>
+                              <PopoverTrigger asChild>
+                                <FormControl>
+                                  <Button className="w-full" variant="outlineInput" childFull onClick={() => setIsCreationDateCalendarOpen(!isCreationDateCalendarOpen)}>
+                                    <div className={calendarInput()}>
+                                      {creationDate ? format(creationDate, 'PPP') : <span>Select a date:</span>}
+                                      <CalendarIcon className={calendarIcon()} />
+                                    </div>
+                                  </Button>
+                                </FormControl>
+                              </PopoverTrigger>
+                              <PopoverContent className={calendarPopover()} align="start">
+                                <Calendar
+                                  mode="single"
+                                  selected={creationDate}
+                                  onSelect={(day) => {
+                                    if (day) {
+                                      form.setValue('creationDate', day)
+                                      setCreationDate(day)
+                                      setIsCreationDateCalendarOpen(false)
+                                    }
+                                  }}
+                                  initialFocus
+                                />
+                              </PopoverContent>
+                            </Popover>
+                          </FormControl>
+                          {form.formState.errors.creationDate && <p className="text-red-500 text-sm">{form.formState.errors.creationDate.message}</p>}
+                        </FormItem>
+                      )}
+                    />
+                  </InputRow>
+
+                  {/* Renewal Date */}
+                  <InputRow className="w-full">
+                    <FormField
+                      control={form.control}
+                      name="renewalDate"
+                      render={({ field }) => (
+                        <FormItem className="w-full">
+                          <FormLabel className="mb-2 flex items-center">
+                            Renewal Date
+                            <TooltipProvider disableHoverableContent>
+                              <Tooltip>
+                                <TooltipTrigger type="button">
+                                  <InfoIcon size={14} className="mx-1 mt-1" />
+                                </TooltipTrigger>
+                                <TooltipContent side="right">
+                                  <p>The date the evidence should be renewed, this is generally the creation date plus one year.</p>
```

Comment:
Let's use Matt's suggested text:
```suggestion
                                  <p>The date the evidence will be re-requested; some evidence may be monthly (e.g. user access reviews), bi-annually, or annually, depending</p>
```